modules/sr/robot/robot.py

```diff
@@ -150,8 +156,9 @@ def sleep(self, secs: float) -> None:
         if secs < 0:
             raise ValueError('sleep length must be non-negative')
 
-        # Ensure the time delay is a valid step increment
-        n_steps = int((secs * 1000) // self._timestep)
+        # Ensure the time delay is a valid step increment, while also ensuring
```

Comment:
You need to handle the secs=0 case specially I think.

Reply:
I was actually planning to leave that to error in the underlying method. Do you think it needs handling other than that?